diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed8f1082..f3742f23 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -608,7 +608,7 @@ add_custom_command(
     TARGET  distclean
 )
 
-include(ClangFormat)
+#include(ClangFormat)
 
 ########################################################################
 # summary
diff --git a/Findlibzmq.cmake b/Findlibzmq.cmake
index 14dd79c4..3f23435a 100644
--- a/Findlibzmq.cmake
+++ b/Findlibzmq.cmake
@@ -28,7 +28,7 @@ find_path (
 
 find_library (
     LIBZMQ_LIBRARIES
-    NAMES zmq
+    NAMES libzmq-v141-mt-4_3_1
     HINTS ${PC_LIBZMQ_LIBRARY_HINTS}
 )
 
