--- Makefile.in	2014-11-20 09:01:54.000000000 +0900
+++ Makefile.in.new	2015-01-05 15:23:42.000000000 +0900
@@ -628,7 +628,7 @@
 # libsbml.pc.
 # -----------------------------------------------------------------------------
 
-base-libs = -lsbml -lstdc++ -lm
+base-libs = -lstdc++ -lm
 
 COMPRESS_CPFLAGS    = ${ZLIB_CPPFLAGS} ${BZ2_CPPFLAGS}
 COMPRESS_LDFLAGS    = ${ZLIB_LDFLAGS} ${BZ2_LDFLAGS}
@@ -653,17 +653,16 @@
 	@echo "libdir=@libdir@"                       >> libsbml.pc
 	@echo "includedir=@includedir@"               >> libsbml.pc
 	@echo ""                                      >> libsbml.pc
+	@echo "Libs: -L\$${libdir} -lsbml"            >> libsbml.pc
+	@echo "Libs.private: ${DEPENDENCY_LIBFLAGS} ${base-libs}" >> libsbml.pc
 ifdef USE_XERCES
-	@echo "Libs: -L\$${libdir} ${DEPENDENCY_LIBFLAGS} ${base-libs}" >> libsbml.pc
 	@echo "Cflags: -I\$${includedir} ${DEPENDENCY_CPPFLAGS}" >> libsbml.pc
 	@echo "Conflicts: xerces-c = 2.6.0" >> libsbml.pc
 endif
 ifdef USE_EXPAT
-	@echo "Libs: -L\$${libdir} ${DEPENDENCY_LIBFLAGS} ${base-libs}" >> libsbml.pc
 	@echo "Cflags: -I\$${includedir} ${DEPENDENCY_CPPFLAGS}"  >> libsbml.pc
 endif
 ifdef USE_LIBXML
-	@echo "Libs: -L\$${libdir} ${DEPENDENCY_LIBFLAGS} ${base-libs}" >> libsbml.pc
 	@echo "Cflags: -I\$${includedir} ${DEPENDENCY_CPPFLAGS}"        >> libsbml.pc
 	@echo "Requires: libxml-2.0 >= 2.6.22" >> libsbml.pc
 endif
