diff --git a/Makefile.am b/Makefile.am
index 1f4619e..31df30f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -73,3 +73,5 @@ EXTRA_DIST = Win32/* \
              tutorial/*.pl \
              tutorial/*.xml
 
+libtool: $(LIBTOOL_DEPS)
+	$(SHELL) ./config.status libtool
diff --git a/configure.ac b/configure.ac
index 936d1a3..f7a1c7e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -63,6 +63,7 @@ dnl Checks for various programs and packages.
 dnl ---------------------------------------------------------------------------
 
 LT_INIT([win32-dll disable-static pic-only])
+AC_SUBST([LIBTOOL_DEPS])
 AC_PROG_CXX
 AC_PROG_CC
 AM_PROG_CC_C_O # deprecated as of automake 1.14
